packages/module/src/Message/UserFeedback/UserFeedback.tsx

```diff
@@ -54,6 +60,20 @@ export interface UserFeedbackProps extends Omit<CardProps, 'onSubmit'>, OUIAProp
   focusOnLoad?: boolean;
   /** Timestamp passed in by Message for more context in aria announcements */
   timestamp?: string;
+  /** Additional props passed to submit button */
+  buttonProps?: ButtonProps;
```

Comment:
Not a blocker since we could make the update if it ever becomes necessary, but `submitButtonProps` if we wanted to future-proof in case we add more buttons to that action group

Reply:
All set @thatblindgeye!